lib/spack/spack/cmd/install.py

```diff
@@ -41,6 +41,8 @@
 from spack.package import PackageBase
 
 description = "build and install packages"
+section = "basic"
+level = "short"
```

Comment:
Would you call `basic usage` something else in that case?  I was trying to group the most important commands under `basic usage`.